### ISSUE TITLE
Context menu: disabled items can be focused and selected with keyboard

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#767]
### Expected behavior

Disabled menu items cannot be focused on or selected with keyboard and have aria-disabled attribute=true. 
### Actual behavior

One can focus and hit return to select disabled items in a menu. Clicking the same item with a mouse does not select it.

**Comment by Blackbaud-PatrickOFriel** [comment]
In [v1.7.0](https://github.com/blackbaud/skyux/releases/tag/v1.7.0)
